pkg/wasm/imagefetcher.go

```diff
@@ -41,6 +42,9 @@ type ImageFetcherOption struct {
 	Username string
 	Password string
 	// TODO(mathetake) Add signature verification stuff.
+
+	// TODO: make this configurable by WasmPlugin
```

Comment:
I am not sure we will ever want to make this first class. Like kubelet, it is just an runtime option for testing/dev purpose.

Reply:
Please think about the private network case, own private register often uses certs signed by an unknown authority(as it is private), which is used more than testing/dev, it usually is used in production, especially in the financial sector.

Reply:
It is not like kubelet, as kubelet will use config from container runtime, but WasmPlugin does not.

Reply:
For those cases, we should add image pull cert to the API, instead of exposing insure options.

Reply:
Unfortunately, now wasm pull secret is also not supported :(